server/src/main/java/org/elasticsearch/rest/ServerlessScope.java

```diff
@@ -22,4 +22,9 @@
 @Target(ElementType.TYPE)
 public @interface ServerlessScope {
     Scope value();
+
+    /**
+     * A value used when restricting a response of a serverless endpoints.
+     */
+    String SERVERLESS_RESTRICTION = "serverless";
```

Comment:
This seems to be unrelated to the PR title?

Reply:
updated the PR